src/task/CleanStoppedSessionTask.ts

```diff
@@ -63,7 +63,7 @@ export default class CleanStoppedSessionTask extends BaseTask {
 
       const tile = this.context.videoTileController.getLocalVideoTile();
       if (tile) {
-        tile.bindVideoStream('', true, null, null, null, null);
+        tile.bindVideoStream('', true, null, null, null, null, '');
```

Comment:
I think passing `null` for external user ID is better. For example, some customers might be using an empty string as a valid external user ID.

Note that the attendee ID (first parameter) can be an empty string since SDK owns it.